models/zillow.py

```diff
@@ -6,27 +6,35 @@
 
 
 class ZillowData(BaseModel):
-    name: Optional[str]
-    type: Optional[str]
+    zpid: Optional[str]
+    property_name: Optional[str]
+    property_type: Optional[str]
+    square_footage: Optional[str]
     number_of_rooms: Optional[str]
-    address: Optional[Any] #remodel this to be of class Address
-    street_name: Optional[Any]
-    state: Optional[str]
-    zip_code: Optional[str]
+    latitude: Optional[str]
+    longitude:Optional[str]
     property_url: Optional[str]
-    square_footage: Optional[str]
-    sales_prices: Optional[str]
+    year_built: Optional[str]
+    heating: Optional[str]
+    cooling: Optional[str]
+    parking: Optional[str]
+    lot_size: Optional[str]
+    price_per_sqft: Optional[str]
+    hoa_dues: Optional[str]
+    sale_price: Optional[str]
 
     def header_fields(self):
         return self.__fields__
 
 
-class Address(BaseModel):
-    """
-    TODO: capture all the fields based on the following data
-    'address': {'@type': 'PostalAddress', '@context': 'http://schema.org', 'streetAddress': '3903 Perkins Rd', 'addressLocality': 'Thompsons Station', 'addressRegion': 'TN', 'postalCode': '37179'}
-    """
-    pass
+class ZillowAddress(BaseModel):
```

Comment:
#question: is this going to be part of ZillowData?

Reply:
Yeah I really wanted to include all of the address data in the csv file.